docs/articles/nunit/writing-tests/attributes/fixturelifecycle.md

```diff
@@ -0,0 +1,38 @@
+---
+uid: fixturelifecycleattribute
+---
+
+# FixtureLifeCycle
+
+Added in **NUnit 3.13.**
+
+The **FixtureLifeCycleAttribute** is used to indicate that an instance for a test fixture or all test fixtures in an assembly should be constructed for each test within the fixture or assembly.

```

Comment:
```suggestion
The `FixtureLifeCycleAttribute` is used to indicate that an instance for a test fixture or all test fixtures in an assembly should be constructed for each test within the fixture or assembly.
```

Suggest using backticks rather than bold as this has some implications for our spell-checking.